pkg/pb/synthetic_monitoring/checks_extra.go

```diff
@@ -220,6 +235,15 @@ func (c *Check) Validate() error {
 		if err := c.Settings.Tcp.Validate(); err != nil {
 			return err
 		}
+
+	case CheckTypeTraceroute:
+		if err := validateHost(c.Target); err != nil {
+			return ErrInvalidHostname
+		}
+
+		if err := c.Settings.Traceroute.Validate(); err != nil {
+			return err
+		}
```

Comment:
I'm wondering whether this code can move up a few lines in this file, so that it lives as close to `case CheckTypePing:` as possible. Ie. just above or below lines 203-210. That way it's easier to eyeball any differences between the two, and hopefully see that they are consistent. 🙏